bindings/flow/fdb_flow.actor.cpp

```diff
@@ -93,333 +94,352 @@ void fdb_flow_test() {
 }
 
 namespace FDB {
-	class DatabaseImpl : public Database, NonCopyable {
```

Comment:
Non-blocking: I'd suggest separate these formatting and convention changes into separate PRs, or at least commits, for maintenance and tracking purpose.

Reply:
Sounds good. I've reverted the formatting changes.